keps/sig-network/20190603-endpointslices/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: EndpointSlice API 
+authors:
+  - "@freehan"
+  - "@robscott"
+owning-sig: sig-network
+reviewers:
+  - "@bowei"
+  - "@thockin"
+  - "@wojtek-t"
+  - "@johnbelamaric"
+approvers:
+  - "@bowei"
+  - "@thockin"
+creation-date: 2019-06-01
+last-updated: 2020-06-01
+status: implementable
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.19"
+
+milestone:
+  alpha: "v1.16"
+  beta: "v1.17"
+  stable: "v1.20"
+
+feature-gate:
+  - name: EndpointSlice
+    components:
+      - kube-apiserver
+      - kube-controller-manager
+  - name: EndpointSliceProxying
+    components:
+      - kube-proxy
```

Comment:
@wojtek-t I couldn't find an example of multiple feature gates, so this is my best guess at what it should look like, open to ideas.

Reply:
Hmm - good catch. We weren't prepared for that TBH.
That sounds reasonable to me, but let me put that into prod-readiness slack if someone has other ideas.

Reply:
Sounds good. I switched the name here to `feature-gates`, let me know if I should change anything here.